docs/en/developer/create-web-component/VUE.md

```diff
@@ -454,7 +457,88 @@ you select _Customize > Add Stuff_.
 
 ## Appendix
 
-### A. Quick rebuild and deploy
+### A. Quick new deployment
+
+You can build and deploy a web component without having to restart the
+uPortal server and re-initialize uPortal data. Once the web component has
+been built and the WebJar dependency has been
+[added to the resource server](#add-webjar-to-resource-server) you can
+deploy it like this.
+
+#### Rebuild and redeploy resource-server
+
+1. Re-build the resource-server so it pulls the new WebJar into its resources.
+
+   ```
+   ./gradlew :overlays:resource-server:build
+   ```
+
+2. Copy the newly-built **resource-server.war** file into the **tomcat/webapps**
+   directory where it will automatically be deployed and overwrite the current one.
+
+   ```
+   cp overlays/resource-server/build/libs/resource-server.war .gradle/tomcat/webapps/
+   ```
+
+#### Import portlet definition or add it in uPortal
+
+It is better, and easier, to create the portlet definition .xml file and
+import it ([Method 1](#method-1-import-portlet-definition)) rather than add
+it in Portlet Administration in uPortal
+([Method 2](#method-2-add-portlet-definition-in-uportal)), but you can do
+it both ways.
+
+##### Method 1: Import portlet definition
+
+1. Create a portlet-definition.xml according to the
+   [instructions](#create-a-portlet-definitionxml) above and
+   [modify it](#edit-portlet-definition) for your web component.
+2. Import the portlet-definition.xml into the uPortal database.
+
+   ```
+   ./gradlew dataImport -Dfile=path/to/portlet-definition.xml
+   ```
+
+3. Clear the uPortal cache in the Cache Manager. In uPortal, navigate to
+   Admin Tools > Cache Administration. Select the **Empty All Caches** button.
+   This causes uPortal to reload data from the database, which will load the
+   portlet definition.
+
+You will now be able to find your portlet in the Customize menu.
+
+##### Method 2: Add portlet definition in uPortal
+
+The problem with this method is that the definition is only temporary. It
+is not included in the uPortal-start project and only resides in the
+database. If you delete or refresh the database, for example with
+`./gradlew portalInit`, the portlet definition is gone.
+
+In uPortal, navigate to Admin Tools > Portlet Administration.
+
+1. Select **Register New Portlet** button.
+2. Select **Advanced CMS** and Continue.
+3. Fill in the Portlet Title, Name, Functional Name, and optionally, Description.
+4. Select **Edit Principals** button then select the **Add to Selection**
+   button for the group(s) that can access the portlet (**Everyone** is the default at
+   the top) and **Save** it.
+5. Select the **Published** radio button under **Lifecycle Management**.
+6. Select the **Save and Configure** button.
+7. In the Content Editor, select the **Source** button and paste the following
+   and substite your component name:
+
+   ```
```

Comment:
```suggestion
   ```html
```

Reply:
Added code hints to all relevant code blocks. Thanks for catching that.